docs/release-notes/NuGet-6.12.md

```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.11.1
+
+NuGet 6.12.1 is available in Visual Studio 17.12.0 and the.NET 9.0.101 SDK.
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* Add property for toggling the to the previous NuGet resolver: RestoreUseLegacyDependencyResolver - [#13700](https://github.com/NuGet/Home/issues/13700)
+
+* Owner profile hyperlinks needed in Details Pane of PM UI - [#13686](https://github.com/NuGet/Home/issues/13686)
+
+* Bubble-up Known Vulnerability Indicators in Solution Explorer for Transitive Packages - [#13636](https://github.com/NuGet/Home/issues/13636)
+
+* Map branch name from sourcelink to RepositoryBranch for NuGet pack - [#13625](https://github.com/NuGet/Home/issues/13625)
+
+* Enable `dotnet nuget why` on non-SDK style projects - [#13576](https://github.com/NuGet/Home/issues/13576)
+
+* NuGetAuditSuppress for packages.config - [#13575](https://github.com/NuGet/Home/issues/13575)
+
+* [Feature] Enable Transitive Dependencies for Solution-level in Visual Studio - [#13216](https://github.com/NuGet/Home/issues/13216)
+
+* Audit security vulnerabilities without adding nuget.org as package source - [#12698](https://github.com/NuGet/Home/issues/12698)
+
+* Roll-out new breaking change process for SDK tools, respect SdkAnalysisLevel - [#13309](https://github.com/NuGet/Home/issues/13309)
+
+### Breaking changes
+
+* NuGetAudit should check transitive packages by default when the .NET 9 SDK is installed [#13293](https://github.com/NuGet/Home/issues/13293)
+
+* Change NuGetAuditMode default to all for non-SDK style projects - [#13584](https://github.com/NuGet/Home/issues/13584)
+
+* [Epic]Deprecate http usage: Promote from warning to error - [#13289](https://github.com/NuGet/Home/issues/13289)
```

Comment:
```suggestion
* Deprecate http usage: Promote from warning to error - [#13289](https://github.com/NuGet/Home/issues/13289)
```

Should we just move this to the What's New?

I think it's really up to us where we want to include it.